build.gradle

```diff
@@ -18,7 +18,7 @@ buildscript {
 }
 
 plugins {
-    id("io.gitlab.arturbosch.detekt").version("1.0.0-RC16")
+    id("io.gitlab.arturbosch.detekt").version("1.6.0")
```

Comment:
Version in `Dependencies` should be also fixed.

Reply:
Done.